modules/vaos/app/controllers/vaos/messages_controller.rb

```diff
@@ -6,8 +6,22 @@ def index
       render json: MessagesSerializer.new(messages[:data], meta: messages[:meta])
     end
 
+    def create
+      response = messages_service.post_message(appointment_request_id, post_params)
+      render json: MessagesSerializer.new(response[:data], meta: response[:meta])
+    end
+
     private
 
+    def appointment_request_id
```

Comment:
I don't care too much about this, but  it feels a bit strange to declare a method to reference `params[:appointment_request_id]` if you're just going to reference it directly on line 26.

Reply:
Will use the defined method. Good catch!